docs/csharp/language-reference/operators/index.md

```diff
@@ -10,11 +11,31 @@ helpviewer_keywords:
 
-In an [expression](../../programming-guide/statements-expressions-operators/expressions.md), operator precedence and associativity determine the order in which the operations are performed. You can use parentheses to change the order of evaluation imposed by operator precedence and associativity.
+The simplest C# expressions are literals (which may be [integer](../builtin-types/integral-numeric-types.md#integer-literals) or [real](../builtin-types/floating-point-numeric-types.md#real-literals) numbers, [characters](../builtin-types/char.md#literals) or [strings](../builtin-types/reference-types.md#the-string-type), [Boolean](../builtin-types/bool.md) `true` or `false`) and simple names (which usually denote variables). With use of operators, you get more complex expressions. Operator [precedence](#operator-precedence) and [associativity](#operator-associativity) determine the order in which the operations in an expression are performed. You can use parentheses to change the order of evaluation imposed by operator precedence and associativity.
+
+In the following code, the right-hand operands of assignments are examples of expressions:
+
+[!code-csharp[expression examples](snippets/Overview.cs#Expressions)]
+
+Additionally, C# provides the following forms of expressions:
+
+- [Interpolated string expressions](../tokens/interpolated.md) that provide convenient syntax to create formatted strings
+- [Lambda expressions](../../programming-guide/statements-expressions-operators/lambda-expressions.md) that allow you to create anonymous functions
+- [Query expressions](../keywords/query-keywords.md) that allow you to use query capabilities directly in C#
+
+Typically, an expression produces a result. A [`void`](../builtin-types/void.md) method invocation is an example of an expression that doesn't produce a result. Any [method invocation](member-access-operators.md#invocation-expression-) is also an example of a *statement expression*, which is an expression that you can use as a statement, that is, in the following form:
+
+```csharp
+statement-expression;
+```
+
+Other statement expressions are [assignment](assignment-operator.md), [await](await.md), [new object instantiation](new-operator.md), [increment](arithmetic-operators.md#increment-operator-), and [decrement](arithmetic-operators.md#decrement-operator---) expressions. When a statement expression is used as a statement, its result, if any, is discarded.
```

Comment:
The items identified as "statement expressions" link to articles that call them operators, which is likely to be confusing.  It might be worth making this list into a bulleted list with an expression example for each.

Reply:
I think reworking these two paragraph per my previous comment may make examples unnecessary. What do you think @tdykstra ?

Reply:
Some parts of your outline after "The only expressions that can be used as statements are:" don't seem obvious to me without examples to explain them:

  - method invocation: this seems obvious
  - `await`: I don't think of "await" as an expression but more like just a keyword you put in front of a method invocation.
  - object creation: I usually think of this as `var x = new Poco();` -- if you discard the result from `new Poco()` what have you accomplished.
  - assignment: same issue here -- assignment I think of as `a = b`, `b` being the expression and `a = b` being the statement; not clear why the later should be called an expression statement
  - pre and post increment and decrement - this one seems obvious

Reply:
@tdykstra @BillWagner thank you for your thoughts and suggestions. However, as this topic looks to be more confusing than I thought, I prefer to remove it at all instead of going into more explanations (which would make the article harder to follow). All in all, expression statements are discussed in the [Statements](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/statements) article. That appears to be a more proper place for them.